woocommerce/wc-admin
# Use static images for nav intro modal

Fixes #6479

Use bundled static images instead of external resources for nav images.

### Screenshots

<img width="694" alt="Screen Shot 2021-03-02 at 3 29 49 PM" src="https://user-images.githubusercontent.com/10561050/109711035-68363700-7b6c-11eb-885b-d17c48ab4751.png">


### Detailed test instructions:

1. Dismiss the welcome modal if you haven't already.
1. Delete the `woocommerce_navigation_intro_modal_dismissed` if it exists.
2. Navigate to the homepage.
3. Note that the images work as expected and are now derived directly from the bundle instead of WooCommerce.com.
